client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -55,43 +119,55 @@ class ThemeEnhancements extends Component {
 		);
 	}
 
+	handleInfiniteDefault() {
+		this.updateInfiniteMode( 'infinite_default' );
+	}
+
+	handleInfiniteButton() {
+		this.updateInfiniteMode( 'infinite_button' );
+	}
+
+	handleInfiniteScroll() {
+		this.updateInfiniteMode( 'infinite_scroll' );
+	}
+
 	renderInfiniteScrollSettings() {
-		const {
-			selectedSiteId,
-			infiniteScrollModuleActive,
-			translate
-		} = this.props;
-		const formPending = this.isFormPending();
+		const { isRequestingSettings, translate } = this.props;
 
 		return (
 			<FormFieldset>
-				<div className="theme-enhancements__info-link-container site-settings__info-link-container">
-					<InfoPopover position={ 'left' }>
-						<ExternalLink href={ 'https://jetpack.com/support/infinite-scroll' } icon target="_blank">
-							{ translate( 'Learn more about Infinite Scroll.' ) }
-						</ExternalLink>
-					</InfoPopover>
-				</div>
+				<FormLabel>
+					<FormRadio
+						name="infinite_mode"
+						value="infinite_default"
+						checked={ 'infinite_default' === this.state.infinite_mode }
+						onChange={ this.handleInfiniteDefault }
+						disabled={ isRequestingSettings }
+						/>
+					<span>{ translate( 'Load more posts using the default theme behavior' ) }</span>
+				</FormLabel>
 
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="infinite-scroll"
-					label={ translate( 'Add support for infinite scroll to your theme' ) }
-					disabled={ formPending }
-					/>
+				<FormLabel>
+					<FormRadio
+						name="infinite_mode"
+						value="infinite_button"
+						checked={ 'infinite_button' === this.state.infinite_mode }
+						onChange={ this.handleInfiniteButton }
+						disabled={ isRequestingSettings }
+						/>
+					<span>{ translate( 'Load more posts in page with a button' ) }</span>
+				</FormLabel>
 
-				<div className="theme-enhancements__module-settings site-settings__child-settings">
-					{
-						this.renderToggle( 'infinite_scroll', ! infiniteScrollModuleActive, translate(
-							'Scroll infinitely (Shows 7 posts on each load)'
-						) )
-					}
-					{
-						this.renderToggle( 'infinite_scroll_google_analytics', ! infiniteScrollModuleActive, translate(
-							'Track each infinite Scroll post load as a page view in Google Analytics'
-						) )
-					}
-				</div>
+				<FormLabel>
+					<FormRadio
+						name="infinite_mode"
+						value="infinite_scroll"
+						checked={ 'infinite_scroll' === this.state.infinite_mode }
+						onChange={ this.handleInfiniteScroll }
+						disabled={ isRequestingSettings }
```

Comment:
This all the other radio inputs probably need to be disabled when saving settings too. 

`disabled={ isRequestingSettings || isSavingSettings }`